pkg/gameserverallocations/controller.go

```diff
@@ -75,6 +85,7 @@ type Controller struct {
 	gameServerGetter       getterv1alpha1.GameServersGetter
 	gameServerLister       listerv1alpha1.GameServerLister
 	allocationPolicyLister multiclusterlisterv1alpha1.GameServerAllocationPolicyLister
+	secretGetter           typedcorev1.SecretsGetter
```

Comment:
should we use a lister here? Getter will always go to the API server, while lister will use cached data

Reply:
good catch!